packages/plugins/ethereum-wallet/polywrap.yaml

```diff
@@ -4,4 +4,4 @@ project:
   name: ethereum-wallet
 source:
   module: ./Cargo.toml
-  schema: ./polywrap.graphql
\ No newline at end of file
+  schema: ./src/polywrap.graphql
```

Comment:
What's the reason for this change? If I'm not mistaken, we've been using the schema in the root (instead of src) more recently. This also IMO, makes more sense since it's where the manifest is and schema (like the manifest) is not a source file. It's also less likely to make IDEs complain

Reply:
no strong reason - actually we were not using it and i just updated the path to make use of it; can def move it to root folder